Cloud-testapp

russo1982 opened the pull request:
# Выполнено ДЗ №6

 - [x] Установка Yandex CLI, Создание инстанса, создания bash скриптов
 - [x] Автоматическое создание инстанса и деплой приложения

## В процессе сделано:
 - Созданы bash скрипты install_ruby.sh , install_mongodb.sh , deploy.sh
 - Создан YAML файл startup.yaml  для создания инстанса и деплоя приложения

## Как запустить проект:
 - Запускаем в Yandex CLI
 - c compute instance create \
  --name reddit-app \
  --hostname reddit-app \
  --memory=2 \
  --create-boot-disk image-folder-id=standard-images,image-family=ubuntu-1604-lts,size=10GB \
  --network-interface subnet-name=default-ru-central1-a,nat-ip-version=ipv4 \
  --metadata serial-port-enable=1 \
  --metadata-from-file user-data=startup.yaml

Файл startup.yaml находится в ветке cloud-testapp

## Как проверить работоспособность:
 - Например, перейти по ссылке http://84.201.132.86:9292       (ссылка будет неактуально. Не могу держать ВМ активным, так как промокода еще нет, а оплачивать дополнительно из своих средств пока не смогу)

## PR checklist
 - [ ] ДЗ №6
 - [ ] Деплой тестового приложения

russo1982 committed:
*.ovpn & setupvpm.sh moved into VPN and scripts created

russo1982 committed:
startup.yaml file added

russo1982 committed:
install_ruby.sh corrected

russo1982 committed:
startup.yaml corrected

russo1982 committed:
startup.yaml corrected

russo1982 committed:
README.md edited

mrgreyves commented on README.md:
```diff
@@ -38,4 +38,23 @@ pritunlVPN Server
+-----------------------------------------------------------------------------------------------------
+ДЗ №6
+
+testapp_IP = 84.201.132.86 (данный IP не статичный поэтому после выгрузки ДЗ инстанс удалю)
```
Комментарий про адрес стоит удалить
Тесты из-за него и валятся =(

mrgreyves reviewed (changes requested):
Здравствуйте!)
Проверил ваше дз
Оставил комментарий
ДЗ возвращаю на доработку

russo1982 committed:
Update README.md

redundant  comments been removed

russo1982 committed:
Update README.md

redundant comments removed. Will keep instance running for you to be able check the HW

mrgreyves reviewed (changes requested):
Здравствуйте!)
Стоит следить за результатами тестов
Нужно починить
ДЗ возвращаю на доработку

russo1982 committed:
README.md edited

russo1982 committed:
README.md o'zgardi

mrgreyves reviewed (approved):
Здравствуйте!)
С дз все ок
Видимо на момент повторных тестов с исправления ВМ уже были потушены
ДЗ принимаю =)
